docs/source/support/release-notes.rst

```diff
@@ -62,6 +62,10 @@ How to migrate:
 - **DAML Assistant**: The assistant handles network failures gracefully.
 
 .. _release-0-12-16:
+- Renamed ``--jdbcurl`` to ``--postgres-backend``. Left ``--jdbcurl`` in place for backwards compat.
```

Comment:
we need only one from this, right? I'd prefer `--sql-backend-jdbcurl`. Also sync with other docs below.

Reply:
done.